feasto_core_clean_arch/interactors/make_order_interactor.py

```diff
@@ -0,0 +1,35 @@
+from feasto_core_clean_arch.exceptions.custom_exceptions import InvalidItemId, InvalidRestaurantId
+from feasto_core_clean_arch.interactors.presenter_interfaces.presenter_interface import PresenterInterface
+from feasto_core_clean_arch.interactors.storage_interfaces.storage_interface import StorageInterface
+
+from typing import List
+
+class MakeOrderInteractor:
+
+    def __init__(self, storage: StorageInterface):
+        self.storage = storage
+
+    def make_order(self,
+                 user_id: int,
+                 presenter: PresenterInterface,
+                 items: List[dict]
+                 ) :
+        for item in items:
+            try:
```

Comment:
Iterative DB calls should never be done, missed below points also

1. try except in iteration is not clean
2. we should not have return statements in iterations, instead we should handle with continue & break